### PR TITLE
Update tag version for glusterfs

### PR DESCRIPTION
**Pull request by jboxman**
This re-opens https://github.com/openshift/openshift-docs/pull/14103.

### CODE REVIEW

**Comment by jboxman** [comment]
So @openshift/team-documentation I'm not sure how to assemble this. Because of the way these files were originally created in relation to each other, there doesn't seem to be any reasonable way to reuse the content.

The `{gluster}` variable refers to the proper name for GlusterFS, and that's defined in `_snippets/` in the root directory. I can't include that in a new file that contains the shared content and then include that file in each of these files.

So I duplicated the content instead of attempting a Byzantium approach.

**Comment by liangxia** [comment]
Check more detail in https://bugzilla.redhat.com/show_bug.cgi?id=1677336#c14 and https://bugzilla.redhat.com/show_bug.cgi?id=1677336#c15

**Comment by jboxman** [comment]
@openshift/team-documentation, PTAL? Thanks!

File: install_config/persistent_storage/topics/glusterfs_images.adoc [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+// * install/configuring_inventory_file.adoc
+// * install_config/persistent_storage/persistent_storage_glusterfs.adoc
+
+// Including expanded macro from: _snippets/glusterfs.adoc
+:gluster: Red Hat Gluster Storage
+
+A valid image tag is required for your deployment to succeed. Replace `<tag>`
+with the version of {gluster} compatible with {product-title} {product-version}
+as described in the
+link:https://access.redhat.com/articles/3403951[interoperability matrix] for the
```
Comment (jarrpa) [inline install_config/persistent_storage/topics/glusterfs_images.adoc]: This link should be changed to point to this page: https://access.redhat.com/articles/3930371

File: install_config/persistent_storage/topics/glusterfs_images.adoc [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+// Valid only for openshift-enterprise
+// Included in the following files:
+// * install/configuring_inventory_file.adoc
+// * install_config/persistent_storage/persistent_storage_glusterfs.adoc
+
+// Including expanded macro from: _snippets/glusterfs.adoc
+:gluster: Red Hat Gluster Storage
+
+A valid image tag is required for your deployment to succeed. Replace `<tag>`
+with the version of {gluster} compatible with {product-title} {product-version}
```
Comment (kalexand-rh) [inline install_config/persistent_storage/topics/glusterfs_images.adoc]: s/compatible/that is compatible

**Review by kalexand-rh — COMMENTED** [review]
If this is rendering the way you expect it to, it LGTM.

File: install_config/persistent_storage/topics/glusterfs_images.adoc (hunk 1 above)
Comment (jboxman) [reply install_config/persistent_storage/topics/glusterfs_images.adoc]: @jarrpa, updated, thanks!

**Comment by jboxman** [comment]
This is verified on the bug for 3.10 and 3.11; merging.

**Pull request merged**